pkg/antctl/raw/proxy/command.go

```diff
@@ -269,6 +269,9 @@ func runE(cmd *cobra.Command, args []string) error {
 	// (eg: port == 0).
 	var l net.Listener
 	if options.unixSocket == "" {
+		if net.ParseIP(options.address).To4() == nil {
+			options.address = fmt.Sprintf("[%s]", options.address)
+		}
```

Comment:
let's not overwrite `options.address` here, you can define a new `address` local variable

Reply:
Updated.